spec/controllers/bookmarks_controller_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-describe BookmarksController do
+describe BookmarksController, bookmark_search: true do
```

Comment:
You can add metadata to any describe/context block, not just the top-level one. In this case you should add `bookmark_search: true` to `context "with caching"`, which are the only tests needing ES.